src/olympia/lib/settings_base.py

```diff
@@ -1128,6 +1128,7 @@ def get_db_config(environ_var, atomic_requests=True):
     'olympia.devhub.tasks.validate_file': {'queue': 'devhub'},
     'olympia.devhub.tasks.validate_upload': {'queue': 'devhub'},
     'olympia.files.tasks.repack_fileupload': {'queue': 'devhub'},
+    'olympia.promoted.tasks.on_stripe_charge_failed': {'queue': 'devhub'},
```

Comment:
I wonder if that's the best queue for it... I suppose it's fine for now - ultimately I hope admins will not rely on this notification only but also periodically visit the Stripe dashboard...

Reply:
Yeah, I don't know either.. I didn't want to create a new queue for that right now, though.